### PR TITLE
Enhancement

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Info.json
    - Rename SP name and description
    - Added "Record Status" picklist
    -  Renamed playbook name from "10 - SP - Outbreak Alert Management Framework" to "10 - SP - Outbreak Response Framework"
    
- Navigation
    -  Renamed Navigation Main menu title from "Outbreak Alert Management" to "Outbreak Management"
    - Renamed navigation submenu title from "Threat Hunt Rules" to "Hunt Rules"

- Outbreak Alerts Module MMD:
    -  Added new fileds "Assigned Date", "Assigned To" and "Record Status" fields
    - Renamed fields  from"Latest Announcement" to "Latest Developments"
 
- Outbreak Alerts Module SVT: Added and rearranged fields 

- Playbooks
     - Renamed playbook collection name from "10 - SP - Outbreak Alert Management Framework" to "10 - SP - Outbreak Response Framework"
    - Added new playbook "Investigate Outbreaks (Type All)" and "Stop Outbreak Alert Record Tracking"
    - Renamed playbook name from "Investigate Outbreak Alert" to "Investigate Outbreak"

### CODE REVIEW

**Review by cs-rohit-khune — APPROVED** [review]
Approved

File: modules/outbreak_alerts/mmd.json [hunk 1]
```diff
@@ -848,7 +1010,7 @@
             "skipSerialization": false,
             "displayName": "{{ latestAnnouncement }}",
```
Comment (cs-suyog-jinturkar) [inline modules/outbreak_alerts/mmd.json:1011]: What about this name?

File: modules/outbreak_alerts/detail-layout.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
                                     "wid": "9a1e5385-5fb4-4c53-ab7a-c2c013fe5c99",
```
Comment (cs-suyog-jinturkar) [inline modules/outbreak_alerts/detail-layout.json:32]: module name rename from outbreak_alerts to outbreaks is not needed?

File: playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreak.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "@type": "Workflow",
     "triggerLimit": null,
-    "name": "Investigate Outbreak Alert",
+    "name": "Investigate Outbreak",
     "aliasName": null,
     "tag": null,
     "description": "Investigate Outbreak Alert on different SIEM tools, remediate and mitigate",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreak.json:7]: Investigate Outbreak Alert --> change needed here too?

File: playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreaks (Type All).json [hunk 4]
```diff
@@ -0,0 +1,141 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Investigate Outbreaks (Type All)",
+    "aliasName": null,
+    "tag": null,
+    "description": "Investigate all active Outbreak Alert on different SIEM tools, remediate and mitigate",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreaks (Type All).json:7]: Investigate all active Outbreak Alert --> alert is needed here?

File: playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json [hunk 5]
```diff
@@ -0,0 +1,117 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Stop Outbreak Alert Record Tracking",
+    "aliasName": null,
+    "tag": null,
+    "description": "Inactivate the selected outbreak alert record",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json:7]: Inactivate  ---> deactivate

File: playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            "status": null,
+            "top": "30",
+            "left": "125",
+            "stepType": "\/api\/3\/workflow_step_types\/f414d039-bb0d-4e59-9c39-a8f1e880b18a",
+            "group": null,
+            "uuid": "c743a37a-ca8a-4eb6-8d19-c857495a7ec4"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Update Outbreak Alert Status Inactive",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json:72]: Update Outbreak Alert Status Inactive. ---> Set Outbreak Alert Status To Inactive

File: playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+            "left": "125",
+            "stepType": "\/api\/3\/workflow_step_types\/b593663d-7d13-40ce-a3a3-96dece928722",
+            "group": null,
+            "uuid": "ab9dfbf4-40fc-4765-92c6-92b865eae2ee"
+        }
+    ],
+    "routes": [
+        {
+            "@type": "WorkflowRoute",
+            "name": "Start -> Update Outbreak Alert Status Inactive",
```
Comment (cs-suyog-jinturkar) [inline playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json:98]: same as above

File: modules/outbreak_alerts/detail-layout.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply modules/outbreak_alerts/detail-layout.json:32]: not required

File: modules/outbreak_alerts/mmd.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply modules/outbreak_alerts/mmd.json:1011]: we mapped fortiguard field name with FSR field

File: playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreak.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreak.json:7]: not required

File: playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreaks (Type All).json (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Response Framework/Investigate Outbreaks (Type All).json:7]: not required

File: playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json (hunk 5 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json:7]: noted

File: playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json (hunk 6 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json:72]: noted

File: playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json (hunk 7 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/10 - SP - Outbreak Response Framework/Stop Outbreak Alert Record Tracking.json:98]: noted

**Pull request merged**